extensions/file/README.md

```diff
@@ -0,0 +1,66 @@
+# File Info Extension Specification
+
+- **Title: File Info** 
+- **Identifier: file**
+- **Field Name Prefix: file**
+- **Scope: Item, Catalog, Collection**
+- **Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+Provides a way to specify file related details such as checksum, data type and size for assets and links in [STAC Items](../../item-spec/item-spec.md), [STAC Catalogs](../../catalog-spec/catalog-spec.md) and [STAC Collections](../../collection-spec/collection-spec.md).
+
+- [Example](examples/sentinel1.json)
+- [JSON Schema](json-schema/schema.json)
+
+## *Link Object* and *Asset Object* fields
+
+The following fields can be used for Links (in the [`Link Object`](../../item-spec/item-spec.md#link-object)) and assets (in the [`Asset Object`](../../item-spec/item-spec.md#asset-object)).
+
+| Field Name         | Type   | Description                                                  |
+| ------------------ | ------ | ------------------------------------------------------------ |
+| file:byte_order | string | The byte order of integer values in the file. One of `big-endian` or `little-endian`. |
+| file:checksum | string | Provides a way to specify file [checksums](#checksums) (e.g. BLAKE2, MD5, SHA1, SHA2, SHA3). The hashes are self-identifying hashes as described in the [Multihash specification](https://github.com/multiformats/multihash) and must be encoded as hexadecimal (base 16) string with lowercase letters. |
+| file:data_type | string | The data type of the file. One of the [data types](#data-types) below. |
+| file:header_size | integer | The header [size](#sizes) of the file, specified in bytes. |
+| file:size | integer | The file [size](#sizes), specified in bytes. |
+
+This extension can OPTIONALLY be used with the [Collection Assets Extension](../collection-assets/README.md).
+File specific details should not be part of the [Item Assets Definition](../item-assets/README.md) in Collections.
+
+### Sizes
+
+Please be aware that the integer values (always unsigned) given for the sizes (especially `file:size`)  may exceed the maximum value for the default integer data type in your environment / programming language. In this specification `integer` specifies a integer number without an upper limit. You might need to use other data types to store the values in. For example, files with a size larger than around 2,14 GB would exceed the maximum value for int32 and in JavaScript `BigInt` could be used then.
+
+### Data Types
+
+The allowed values for `file:data_type` are:
+
+- `int8`: 8-bit integer
+- `int16`: 16-bit integer
+- `int32`: 32-bit integer
+- `int64`: 64-bit integer
+- `uint8`: unsigned 8-bit integer (common for 8-bit RGB PNG's)
+- `unit16`: unsigned 16-bit integer
```

Comment:
```suggestion
- `uint16`: unsigned 16-bit integer
```

Reply:
Thank you, included it in my last commit!